src/coreclr/inc/profilepriv.inl

```diff
@@ -84,23 +85,24 @@ inline void ProfilerInfo::ResetPerSessionStatus()
 inline void ProfilerInfo::Init()
 {
     curProfStatus.Init();
-    dwProfilerEvacuationCounter = 0;
     ResetPerSessionStatus();
     inUse = FALSE;
 }
 
+inline HRESULT AnyProfilerPassesConditionHelper(BOOL *pAnyPassed, VolatilePtr<EEToProfInterfaceImpl> profInterface)
```

Comment:
The arguments should not be `VolatilePtr`. It does not make sense to apply volatile access to the argument since the argument value is copied onto thread stack where no other thread can access it.